sdk/dotnet/Pulumi/PublicAPI.Shipped.txt

```diff
@@ -172,6 +172,8 @@ Pulumi.ResourceTransformationResult
 Pulumi.ResourceTransformationResult.Args.get -> Pulumi.ResourceArgs
 Pulumi.ResourceTransformationResult.Options.get -> Pulumi.ResourceOptions
 Pulumi.ResourceTransformationResult.ResourceTransformationResult(Pulumi.ResourceArgs args, Pulumi.ResourceOptions options) -> void
+Pulumi.Serialization.IResourcePackage
+Pulumi.Serialization.IResourcePackage.Construct(string name, string type, System.Collections.Generic.IDictionary<string, object> args, string urn) -> Pulumi.Resource
```

Comment:
We likely want to keep all user-facing API in the root `Pulumi` namespace

Reply:
Fixed.